File: api-examples/cat/search/java/multiAttribiute.java
OkHttpClient client = new OkHttpClient().newBuilder()
  .build();
Request request = new Request.Builder()
  .url("https://pudx.catalogue.iudx.org.in/catalogue/v1/search?attribute-name=(tags,resourceServerGroup)&attribute-value=((streetlight,flood),(pudx-resource-server/flood-sensor))")
  .method("GET", null)
  .build();
Response response = client.newCall(request).execute();

File: api-examples/cat/search/js/multiAttribiute.js
var settings = {
  "url": "https://pudx.catalogue.iudx.org.in/catalogue/v1/search?attribute-name=(tags,resourceServerGroup)&attribute-value=((streetlight,flood),(pudx-resource-server/flood-sensor))",
  "method": "GET",
  "timeout": 0,
};

$.ajax(settings).done(function (response) {
  console.log(response);
});
